doc/sql_guide.md

```diff
@@ -0,0 +1,366 @@
+  ```
+  ```
+  ======================================  Welcome to Interactive Mode ======================================  
+  Usage: 
+      help         print the usage of interactive
+      run <arg>    the path of case file to run
+      run_cci <arg>    the path of case file to run_cci by cci driver
+      quit         quit interactive mode 
+
+  For example:
+      run .                                                 #run the current directory cases
+      run ./_001_primary_key/_002_uniq                      #run the cases which are based on the relative path
+      run test.sql                                          #run the test.sql file
+      run /home/user1/dailyqa/trunk/scenario/sql/_02_object #run the cases which are based on the absolute path
+      run_cci ./_001_primary_key/_002_uniq                  #run the cases which are based on the relative path
+      run_cci ./_001_primary_key/_002_uniq/test.sql         #run the cases file
+  sql>run .
+  ```
+
+  ![run_test](./sql_image/ctp_run_test.png)  
+
+  Alternatively, you can use the webconsole of CTP to check results.
+    
+# 3. Regression Test Deployment  
+## 3.1 Deployment overview    
+  No|role|user|ip|hostname|QUEUE_NAME|run_name
+  :---:|:--:|:---:|:---:|:---:|:---:|:---:
+  1|Test node|sql1|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+  2|Test node|sql2|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+  3|Test node|sql3|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+  4|Test node|sqlbycci|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_CCI_LINUX_GIT|run_sql_by_cci
+  5|Test node|sql|192.168.1.77|func02|QUEUE_CUBRID_QA_SQL_PERF_LINUX|run_sql
+  6|Test node|qa|192.168.1.161|winfunc01|QUEUE_CUBRID_QA_SQL_WIN64|run_sql 
+## 3.2 Installation 
+* Install reference [CTP_install_guide](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#3-install-ctp-as-regression-test-platform) 
+* Check out test cases
+  ```
+  cd ~/
+  git clone https://github.com/CUBRID/cubrid-testcases.git  
+  ```
+* Modify .bash_profile
+  ```
+  export CTP_HOME=$HOME/CTP
+  export CTP_SKIP_UPDATE=0
+  export CTP_BRANCH_NAME=develop
+  . ~/.cubrid.sh
+  export PATH=$HOME/CTP/bin:$HOME/CTP/common/script:$PATH:$HOME/.local/bin:$PATH:$HOME/bin
+
+  export TZ='Asia/Seoul'
+  export LC_ALL=en_US
+  ```
+* Add quick start script file    
+  * Linux Queue configure 
+    touch start_test.sh  
+    ```
+    stop_consumer.sh 
+    prefix=`date "+%Y%m%d%H%M%S"`
+    cp nohup.out nohup.out.$prefix
+    echo "" > nohup.out
+    nohup start_consumer.sh -q [QUEUE_NAME] -exec [run_name] &
+    ```       
+  * Windows Queue configure   
+      touch start_test.sh
+      ```
+      rm nohup.out
+      nohup start_consumer.sh -q [QUEUE_NAME] -exec [run_name] &
+      ```
+  * sh start_test.sh  
+
+* send test message  
+  login message@192.168.1.91 and send test message like:  
+  ```
+  sender.sh  [QUEUE_NAME]  [CI_BUILD] [Category] default
```

Comment:
Directly show actual command one by one.
Otherwise readers can't give exact queue name and other parameters.